src/ui/components/ShowMoreCard/index.js

```diff
@@ -29,15 +29,13 @@ export class ShowMoreCardBase extends React.Component {
   }
 
   componentWillReceiveProps() {
-    this.truncateToMaxHeight(ReactDOM.findDOMNode(this.contents));
+    this.setState({ expanded: true }, () => {
+      this.truncateToMaxHeight(ReactDOM.findDOMNode(this.contents));
```

Comment:
While we're here, should we still be using `ReactDOM.findDOMNode`? I forget if we can do this with refs or not but maybe we could?